das_api/src/api/api_impl.rs

```diff
@@ -313,6 +313,7 @@ impl ApiContract for DasApi {
             before,
             after,
             json_uri,
+            collection_metadata,
```

Comment:
rename to "show_collection_metadata" 

long-term i think nesting this would be best, e.g.

```
displayOptions: { show_collection_metadata: true }
```

but dont need to do for now, i'd like to ask customers for what they prefer